src/Topshelf.FSharp.Example/Topshelf.FSharp.Example.fsproj

```diff
@@ -64,7 +65,19 @@
   <PropertyGroup>
     <MinimumVisualStudioVersion Condition="'$(MinimumVisualStudioVersion)' == ''">11</MinimumVisualStudioVersion>
   </PropertyGroup>
-  <Import Project="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets" Condition=" Exists('$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets')" />
+  <Choose>
+    <When Condition="'$(VisualStudioVersion)' == '11.0'">
+      <PropertyGroup>
+        <FSharpTargetsPath>$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>

```

Comment:
Yeah I can fix if I get more time later. On my to a wedding so limited time this weekend. 

Opened it in VS 15, that's probably why I got that version.

-----Original Message-----
From: "Henrik Feldt" <notifications@github.com>
Sent: ‎2015-‎06-‎26 19:26
To: "haf/Topshelf.FSharp" <Topshelf.FSharp@noreply.github.com>
Cc: "Tomas Jansson" <tomas.jansson@gmail.com>
Subject: Re: [Topshelf.FSharp] "More" fluent API and more true with F# (#1)

In src/Topshelf.FSharp.Example/Topshelf.FSharp.Example.fsproj:
> @@ -64,7 +65,19 @@
>    <PropertyGroup>
>      <MinimumVisualStudioVersion Condition="'$(MinimumVisualStudioVersion)' == ''">11</MinimumVisualStudioVersion>
>    </PropertyGroup>
> -  <Import Project="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets" Condition=" Exists('$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets')" />
> +  <Choose>
> +    <When Condition="'$(VisualStudioVersion)' == '11.0'">
> +      <PropertyGroup>
> +        <FSharpTargetsPath>$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
Should be v3.1
—
Reply to this email directly or view it on GitHub.